<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>995</width>
    <height>935</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="topContentWidget">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="segmentation_properties">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Segmentation strategies</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QRadioButton" name="conditionRButton">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Conditon-wise</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>- Split the signal into epochs  based on experimental conditions (e.g., rest, task).</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QRadioButton" name="eventRButton">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Event-wise</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>- Segment the signal using event markers.  Requires you to select at least one target condition or event type.</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Please, </string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="trials">
        <item>
         <widget class="QLabel" name="trialLabel">
          <property name="toolTip">
           <string>Epochs length in samples</string>
          </property>
          <property name="text">
           <string>Trial length (ms): </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="trialBox">
          <property name="font">
           <font>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhDigitsOnly</set>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
          <property name="keyboardTracking">
           <bool>true</bool>
          </property>
          <property name="minimum">
           <number>1000</number>
          </property>
          <property name="maximum">
           <number>100000</number>
          </property>
          <property name="singleStep">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="trialstrideLabel">
          <property name="text">
           <string>Stride (%):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="trialstrideBox">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="winLabel_1">
          <property name="toolTip">
           <string>Temporal window in ms of the epoch. For example, w_epoch_t = [0, 1000] takes the epoch form 0 ms to 1000 ms after each onset (0 ms represents the onset).</string>
          </property>
          <property name="text">
           <string>Window interval (ms): from </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="winBox_1">
          <property name="font">
           <font>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="minimum">
           <number>-100000</number>
          </property>
          <property name="maximum">
           <number>100000</number>
          </property>
          <property name="singleStep">
           <number>100</number>
          </property>
          <property name="value">
           <number>-100</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="winLabel_2">
          <property name="text">
           <string>to</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="winBox_2">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="minimum">
           <number>-100000</number>
          </property>
          <property name="maximum">
           <number>100000</number>
          </property>
          <property name="singleStep">
           <number>100</number>
          </property>
          <property name="value">
           <number>400</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="signal_markers">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Signal markers</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="available_conditions">
        <item>
         <widget class="QWidget" name="conditionWidget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="availableconditionsLabel">
             <property name="text">
              <string>Available conditions</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListView" name="conditionList"/>
           </item>
           <item>
            <widget class="QLabel" name="conditionLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>3</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Conditions found: </string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="available_events">
        <item>
         <widget class="QWidget" name="eventWidget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="availableeventsLabel">
             <property name="text">
              <string>Available events</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListView" name="eventList"/>
           </item>
           <item>
            <widget class="QLabel" name="eventLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>3</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Events found: </string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Epoch parameters</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QCheckBox" name="normCBox">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Normalization</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="normLabel">
          <property name="font">
           <font>
            <italic>false</italic>
           </font>
          </property>
          <property name="text">
           <string>- Over condition segment</string>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <property name="indent">
           <number>-8</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="zscoreRButton">
          <property name="text">
           <string>Z-score</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="dcRButton">
          <property name="text">
           <string>DC</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="baselineLabel_1">
          <property name="toolTip">
           <string>Temporal window in ms of the baseline. For example, w_baseline_t = [-500, 100] takes the baseline from -500 ms before each onset to 100 ms after each onset (0 ms represents the onset). This chunk of signal is used to normalize the epoch, if applicable.</string>
          </property>
          <property name="text">
           <string>Baseline window (ms): from</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="baselineCBox_1">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="minimum">
           <number>-100000</number>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
          <property name="value">
           <number>-100</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="baselineLabel_2">
          <property name="text">
           <string>to</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="baselineCBox_2">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="minimum">
           <number>-100000</number>
          </property>
          <property name="maximum">
           <number>1000000</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QCheckBox" name="thresCBox">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Thresholding</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="thresLabel">
          <property name="text">
           <string>- Automatic rejection of noisy epochs</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="threskLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>Standard deviation multiplier to calculate threshold</string>
          </property>
          <property name="text">
           <string>σ</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="threskBox">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>2.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="threskLabelaux">
          <property name="font">
           <font>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string>0%</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="thressampLabel">
          <property name="toolTip">
           <string>Minimum number of samples that have to be over the threshold in each epoch to be discarded</string>
          </property>
          <property name="text">
           <string>Samples</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="thressampBox">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
          <property name="value">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="threschanLabel">
          <property name="text">
           <string>Channels</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="threschanBox">
          <property name="toolTip">
           <string>Minimum number of channels that have to have n_samples over the threshold in each epoch to be discarded</string>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>999</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="threshelButton">
          <property name="text">
           <string>Help</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="averageCBox">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Average epochs</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="averageLabel">
          <property name="text">
           <string>- Average all the epochs of each condition </string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Resampling epochs</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QCheckBox" name="resampleCBox">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Resample</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="resmapleLabel">
        <property name="text">
         <string>- Adjust epochs sampling rate </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="newfsLabel">
        <property name="text">
         <string>Target sampling frequency</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="resamplefsBox">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
        <property name="value">
         <number>150</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
